js/main.js

```diff
@@ -16,12 +16,12 @@ var $dealerHand = document.querySelector('.dealer-hand');
 
 // global variables
 
-var drawnCards = null;
+var xhr;
```

Comment:
You shouldn't make `xhr` global like this. Generally people do this because they want access to the response in a different function. That just means you should add a parameter to the function, and pass that value you need as an argument. It might mean changing this:
`xhr.addEventListener('load', loadCallback);`
to this:
```
xhr.addEventListener('load', function(){
  loadCallback(xhr.response)
});
```